.circleci/config.yml

```
@@ -219,10 +219,16 @@ jobs:
                 --output-file "$CIRCLE_TEST_REPORTS/eslint/results.xml" \
                 $FILES_TO_LINT
             fi
```

We need to make sure test results are stored for this job:

```suggestion
             fi
      - store-artifacts-and-test-results

```